wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/wsworkunits/WUInfoRequestWrapper.java

```diff
@@ -1,683 +1,685 @@
-    {
-        this.includeResourceURLs = includeResourceURLs;
-    }
-
-    /**
-     * Gets the include ECL.
-     *
-     * @return the includeECL
-     */
-    public Boolean getIncludeECL()
-    {
-        return includeECL;
-    }
-
-    /**
-     * Sets the include ECL.
-     *
-     * @param includeECL
-     *            the includeECL to set
-     */
-    public void setIncludeECL(Boolean includeECL)
-    {
-        this.includeECL = includeECL;
-    }
-
-    /**
-     * Gets the include helpers.
-     *
-     * @return the includeHelpers
-     */
-    public Boolean getIncludeHelpers()
-    {
-        return includeHelpers;
-    }
-
-    /**
-     * Sets the include helpers.
-     *
-     * @param includeHelpers
-     *            the includeHelpers to set
-     */
-    public void setIncludeHelpers(Boolean includeHelpers)
-    {
-        this.includeHelpers = includeHelpers;
-    }
-
-    /**
-     * Gets the include allowed clusters.
-     *
-     * @return the includeAllowedClusters
-     */
-    public Boolean getIncludeAllowedClusters()
-    {
-        return includeAllowedClusters;
-    }
-
-    /**
-     * Sets the include allowed clusters.
-     *
-     * @param includeAllowedClusters
-     *            the includeAllowedClusters to set
-     */
-    public void setIncludeAllowedClusters(Boolean includeAllowedClusters)
-    {
-        this.includeAllowedClusters = includeAllowedClusters;
-    }
-
-    /**
-     * Gets the include total cluster time.
-     *
-     * @return the includeTotalClusterTime
-     */
-    public Boolean getIncludeTotalClusterTime()
-    {
-        return includeTotalClusterTime;
-    }
-
-    /**
-     * Sets the include total cluster time.
-     *
-     * @param includeTotalClusterTime
-     *            the includeTotalClusterTime to set
-     */
-    public void setIncludeTotalClusterTime(Boolean includeTotalClusterTime)
-    {
-        this.includeTotalClusterTime = includeTotalClusterTime;
-    }
-
-    /**
-     * Gets the suppress result schemas.
-     *
-     * @return the suppressResultSchemas
-     */
-    public Boolean getSuppressResultSchemas()
-    {
-        return suppressResultSchemas;
-    }
-
-    /**
-     * Sets the suppress result schemas.
-     *
-     * @param suppressResultSchemas
-     *            the suppressResultSchemas to set
-     */
-    public void setSuppressResultSchemas(Boolean suppressResultSchemas)
-    {
-        this.suppressResultSchemas = suppressResultSchemas;
-    }
-
-    /**
-     * Gets the thor slave IP.
-     *
-     * @return the thorSlaveIP
-     */
-    public String getThorSlaveIP()
-    {
-        return thorSlaveIP;
-    }
-
-    /**
-     * Sets the thor slave IP.
-     *
-     * @param thorSlaveIP
-     *            the thorSlaveIP to set
-     */
-    public void setThorSlaveIP(String thorSlaveIP)
-    {
-        this.thorSlaveIP = thorSlaveIP;
-    }
-}
+package org.hpccsystems.ws.client.wrappers.wsworkunits;
```

Comment:
It could...but that would mean all of the other files out in github with \n line endings would be uploaded as \r\n. I did this one in the past and royally hosed a repo, so I'm a bit leery of doing so.   

I can submit a separate pre- PR in that just pushes this class with the corrected line endings and no other changes; then this current PR won't show them.